common.blocks/add-form/add-form.js

```diff
@@ -37,6 +37,16 @@ modules.define('add-form', [
             return this;
         },
 
+        getRenderedContent: function() {
+            return this._editor.render();
+        },
+
+        clear: function() {
```

Comment:
I still want to try and rewrite it to generic `form` block from `bem-bricks` so decided not to think to here :)